src/wp-includes/functions.php

```diff
@@ -7635,3 +7633,72 @@ function is_php_version_compatible( $required ) {
 function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
+/**
+ * Wraps inline JavaScript in `<script>` tags.
+ *
+ * Enables to inject attributes in the `<script>` tag via the `script_attributes` filter.
+ * Automatically injects type attribute if needed.
+ *
+ * @param array     $attributes `<script>` tag attributes.
+ * @param bool      $double_quotes  Optional. Uses double quotes `"` to wrap attribute values if true, single quotes `'` otherwise.
+ * @param bool      $echo       Optional. Prints to the page if true, returns the value otherwise.
+ * @param string    $content    Optional. Code to be wrapped around `<script>` tags.
+ * @return string|null  Inline JavaScript code wrapped around <script> tags if $echo is true, null otherwise.
+ */
+function wp_script( $attributes, $double_quotes = true, $echo = true, $content = '' ) {
```

Comment:
Revisiting this, I think the new functions should get more distinct names. WP already has several functions dealing with the enqueuing scripts functionality from which these two functions are quite different since the print a script tag right away. I think it would be better to call this one `wp_print_script_tag()`.

Reply:
What do you think of `wp_print_script_loader_tag()` instead? I think it indicates the purpose more clearly.

Reply:
Hmm, what would `loader` clarify here? Could you explain your thoughts a bit more?

Reply:
It is also used in the name of [this](https://developer.wordpress.org/reference/hooks/script_loader_tag/) filter to show that the script tag loads an external resource through the `src` attribute.

Reply:
Changed it to `wp_print_script_tag()`